content/en/docs/getting-started-minikube.md

```diff
@@ -0,0 +1,181 @@
++++
```

Comment:
Do we really need this guide over here? I guess we should not keep third party installation steps in our repo, we should just provide a link to other official website for third party installation

Reply:
deleted